generator/src/Data/Shortcut/Modifiers.purs

```diff
@@ -0,0 +1,96 @@
+{-
+Copyright 2015 SlamData, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module Data.Shortcut.Modifiers where
+
+import Prelude
+import Data.Char (fromCharCode)
+import Data.KeyboardEvent (KeyboardEvent())
+import Data.Shortcut.Platform (Platform(..))
+import Data.String (joinWith, fromChar)
+
+data KeyModifiers = KeyNone | KeyMod
+
+data CodeModifiers = None | Alt | Shift | Mod | AltShift | AltMod | ShiftMod | AltShiftMod
```

Comment:
Actually I'm not sure about `Mod` it looks like pure fabrication that should be implemented on next level of abstraction.